### ISSUE TITLE
Is Odd blur normal when performance drops?

### ISSUE THREAD

**Issue by PeckingWater** [jrbudda/Vivecraft_112#211]
I'm having a problem where my hands and in more severe occasions the world around me start to have a strange blur/distortion and stretched effect. I got rid of all my mods and shaders and booted up Minecraft with just vivecraft, issue still occurs. Interestingly, the issue occurs when i intentionally set performance low (setting render distance to about 24). I have a 64gb ram computer with a 1660 super GPU and an overall very good computer so i dont think thats the issue. My vivecraft version is vivecraft-1.19.2-1.1.14-forge. Is there a setting to stop this weird effect? Do i just need to stick to very low render distance? If an image is needed i can attach it. Any help would be very appreciated Thank You!

**Comment by fayer3** [comment]
yes that is normal, that is the quests/oculus spacewarp kicking in
also you are on the wrong bug tracker
